addons/product/report/product_product_templates.xml

```diff
@@ -21,12 +21,8 @@
                             <div class="o_label_extra_data">
                                 <t t-out="extra_html"/>
                             </div>
-                            <t t-if="product.is_product_variant">
-                                <strong class="o_label_price" t-field="product.lst_price" t-options="{'widget': 'monetary', 'label_price': True}"/>
-                            </t>
-                            <t t-else="">
-                                <strong class="o_label_price" t-field="product.list_price" t-options="{'widget': 'monetary', 'label_price': True}"/>
-                            </t>
+                            <strong class="o_label_price" t-out="pricelist._get_product_price(product, 1, False)"
+                                    t-options="{'widget': 'monetary', 'display_currency': pricelist.currency_id or product.currency_id, 'label_price': True}"/>
```

Comment:
The label is printed with the current company currency rate applied, not the product currency, thus the wrong currency is shown.

Here, the product price is 150 ($, as the main company), so, without pricelist, when I print the label, the price is converted to the company currency, here €, but the label is printed with the $ symbol.

![image](https://github.com/odoo/odoo/assets/69820391/5a31edab-caf8-4c51-9cc5-de59319e4549)

The fact that the product currency is the one from the main company can be tricky. Maybe add a help message to the field, or a span to the wizard when there is no pricelist shown, to explain that the company currency will be used instead of the product one?

cc: @dalaOdoo